pom.xml

```diff
@@ -31,9 +30,8 @@
         <maven.compiler.target>1.8</maven.compiler.target>
 
         <bitbucket.version>5.5.0</bitbucket.version>
-        <bitbucket.data.version>${bitbucket.version}</bitbucket.data.version>
-        <atlassian-sal-api.version>3.1.0</atlassian-sal-api.version>
-        <amps.version>6.3.7</amps.version>
+        <bitbucket.test.version>${bitbucket.version}</bitbucket.test.version>
```

Comment:
This allows testing on different versions of Bitbucket Server, while retaining your current API baseline.